zion/contracts/goerli/addresses/council.json

```diff
@@ -0,0 +1 @@
+{"councilnft": "0xae599a7c5f2cad3acd773e1931045384d70cb4e6"}
```

Comment:
Should these coming in from the dendrite update? Won’t we be running a script in this repo to update them?

Reply:
The script runs in the mono repo as part of the contract publish step. We don't have a way to publish them from mono repo to the dendrite fork. So I'm manually copying them over.